core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -48,6 +51,31 @@ private[spark] object JettyUtils extends Logging {
   val SPARK_CONNECTOR_NAME = "Spark"
   val REDIRECT_CONNECTOR_NAME = "HttpsRedirect"
 
+  val snappyDataRealm = "SnappyDataPulse"
+  val snappyDataRoles = Array("user")
```

Comment:
It is scala class and in scala if no modifier is specified then that means public. :-)

Scala does not have public keyword for defining public access modifier.